new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -41,11 +54,84 @@ public void testCreatingDomain() {
         "Failed to create unique namespace due to ApiException");
     logger.info(String.format("Got a new namespace called %s", namespace));
 
+    // create a persistent volume claim
+    final String storageClassName = domainUID + "-weblogic-domain-storage-class";
+    final String pvcName = domainUID + "-pvc"; // name of the persistent volume claim
+    final String pvName = domainUID + "-pv"; // name of the persistent volume
+    HashMap<String, String> labels = new HashMap();
```

Comment:
Sankar, considering that your code is going to be the sample that everyone copies, I think it's worth taking the time to simplify and shorten this code as much as possible.  Unless you are going to use constants later, it would be better to not create these variables and just use the values inline below.  For instance, you don't need to create a labels Map.  V1ObjectMeta has putLabelsItem() that you can chain.

Reply:
Thanks for that review Ryan. Please see if the latest changes are good.